features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/semantics/TextSemanticsNodeMapper.kt

```diff
@@ -26,11 +26,17 @@ internal open class TextSemanticsNodeMapper(
         parentContext: UiContext,
         asyncJobStatusCallback: AsyncJobStatusCallback
     ): SemanticsWireframe {
+        val wireframes = arrayListOf<MobileSegment.Wireframe>()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ what about the legacy `TextView` in declarative UI. Are we handling the overflow correctly there ? I don't remember honestly.

Reply:
Yes, In imperative UI it is correctly handled in this [PR](https://github.com/DataDog/dd-sdk-android/pull/2279)